scripts/windows/test/Setup-Env.ps1

```diff
@@ -86,19 +100,22 @@ If ($osEdition -eq "IoTUAP")    # Windows IoT Core - update iotedge
             Remove-Item $FileName
         }        
         
-        Write-Host "Attempt to update $serviceName..."
-        try {
-            # triggers reboot
-            Update-IoTEdge -ContainerOs Windows -OfflineInstallationPath $IoTEdgedArtifactFolder
-        }
-        catch {
-            $testExitCode = [String]::Format("0x{0:X}", $LastExitCode)
-            Write-Host "Update attempt unsuccessful (error code $testExitCode)."
-            if ($LastExitCode -eq -2145877235) {
-                Write-Host "A newer version is already installed on the device."
+        if ($AttemptUpdate) {
```

Comment:
in what scenario,  we clean up container and config file, but don't want to update/install iotedge service?

Reply:
The E2E test stage runs 10 different tests one after another. It is only meaningful to attempt update before the very first test step (e.g. TempSensor), not do updates between the tests (e.g. TempSensor and TempFilte).
The E2E test stage for Windows Pro does cleanup between tests by uninstalling/reinstalling the iotedge service - that's my understanding.
We'd like to do cleanup in between the test steps on Windows IoT Core, so that stage behaves as close as possible to Windows Pro.

Reply:
If you use attemptUpdate option to avoid install/update of iotedge service, you should not remove config.yaml as well.

Reply:
On Windows IoT Core I still need to clean up _config.yaml_ file in between tests otherwise _Run-E2ETest.ps1_ fails when run with parameter _BypassEdgeInstallation_.